CFC_WebApp/clients/leaderboard/result_template.html

```diff
@@ -29,9 +29,13 @@
         <body onLoad='showscores()'></body>
     </div>
     <script>
+        var client = new WindowsAzure.MobileServiceClient(
```

Comment:
So result_template is currently loaded from the server and has the results encoded in it. We want to switch to a model in which we read the data from the local database that has been synced there earlier. I don't see that change in here. I hope that is in the phone code (I'm going to review that next), but in that case, you should revert the changes to this code since we don't want to change it....